### PR TITLE
tasks: add new `linkerd-smi` getting started guide

### PR DESCRIPTION
**Pull request by Pothulapati**
This PR adds a new task to get started with the
`linkerd-smi` extension where we go through the
installation and try it with a sample application

Signed-off-by: Tarun Pothulapati <tarunpothulapati@outlook.com>

### CODE REVIEW

**Comment by cpretzer** [comment]
@Pothulapati is this ready to be merged after review? Or do we need to wait for the `2.11` release?

File: linkerd.io/content/2.10/tasks/linkerd-smi.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+Install the SMI extension CLI binary by running:
+
+```bash
+curl -sL https://linkerd.github.io/linkerd-smi/install | sh
+```
+
+Alternatively, you can download the CLI directly via the [releases page](https://github.com/linkerd/linkerd-smi/releases).
+
+The first step is installing the Linkerd-Jaeger extension onto your cluster.
```
Comment (cpretzer) [inline linkerd.io/content/2.10/tasks/linkerd-smi.md:46]: Should this be `Linkerd-SMI`?

**Review by cpretzer — CHANGES_REQUESTED** [review]
One small change to remove the reference to `Linkerd-Jaeger` and this is good to go.

I walked through the steps and everything looks good.

One thing I found useful during my testing was to look at the logs for the `smi-adaptor` pod, so an example of that might be useful, but not necessary

**Comment by Pothulapati** [comment]
@cpretzer Updated!

> @Pothulapati is this ready to be merged after review? Or do we need to wait for the 2.11 release?

Yep, Should be ready to be merged after review!

**Review by cpretzer — APPROVED** [review]

**Pull request merged**